environment.yml

```diff
@@ -4,34 +4,33 @@
 name: svgdigitizer-build
 channels:
   - conda-forge
-  - defaults
 dependencies:
   - astropy>=5,<6
-  - black
-  - click=8.0.3
+  - black>=23,<24
+  - click>=8,<9
```

Comment:
why pin these versions? I thought we always wanted the most recent version to validate our code. The same applies to `isort` which is not pinned.

Reply:
black changes how they are formatting things with version upgrades. If we don't pin, the version jumps randomly in some PR and that PR is going to fail for no good reason (just because formatting changed.) That's annoying so it's maybe better to make this version bump explicitly in a separate PR when there is a new black.

Are you also asking about click? I don't know. It's the same pin we have for the project. Probably because we are not compatible with click 9?

Reply:
yes sorry for the click. I selected the wrong lines. :)
thanks for the info on black and pylint.